drivers/virtio/virtio-gpu.c

```diff
@@ -528,7 +535,8 @@ static int virtio_gpu_probe(FAR struct virtio_device *vdev)
   priv->stride = priv->xres * VIRTIO_GPU_BPP >> 3;
   priv->fblen  = priv->stride * priv->yres;
 
-  priv->fbmem  = (FAR uint8_t *)virtio_zalloc_buf(vdev, priv->fblen, 16);
+  priv->fbmem = (FAR uint8_t *)virtio_zalloc_buf(vdev, priv->fblen,
```

Comment:
don't need change if vm_map_region support the unaligned address.

Reply:
and here too

Reply:
If fbmem is not paged aligned, then the kernel memory region before fbmem will be exposed to user space, which looks like a security hole.